src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const swap = {};
+  let hasDublicate = false;
+
+  for (const key in robot) {
+    if (swap[robot[key]]) { //
```

Comment:
```suggestion
    if (swap[robot[key]]) {
```
do not keep useless comments